opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AisStagesCommonUtil.java

```diff
@@ -29,16 +31,24 @@
     public static final String SESSION_PASSWORD = "qwerty";
     public static final String ANTON_BRUECKNER = "anton.brueckner";
     public static final String MAX_MUSTERMAN = "max.musterman";
+    public static final String COMPUTE_IP_ADDRESS = "false";
+    public static final String IP_ADDRESS = "1.1.1.1";
 
     public static RequestSpecification withDefaultHeaders(String fintechUserId) {
+        return withHeadersWithoutIpAddress(fintechUserId)
+                .header(COMPUTE_PSU_IP_ADDRESS, COMPUTE_IP_ADDRESS)
+                .header(PSU_IP_ADDRESS, IP_ADDRESS);
+    }
+
+    public static RequestSpecification withHeadersWithoutIpAddress(String fintechUserId) {
         return RestAssured
                 .given()
-                    .header(AUTHORIZATION, DEFAULT_AUTHORIZATION)
-                    .header(BANK_ID, SANDBOX_BANK_ID)
-                    .header(FINTECH_REDIRECT_URL_OK, FINTECH_REDIR_OK)
-                    .header(FINTECH_REDIRECT_URL_NOK, FINTECH_REDIR_NOK)
-                    .header(FINTECH_USER_ID, fintechUserId)
-                    .header(SERVICE_SESSION_PASSWORD, SESSION_PASSWORD)
-                    .header(X_REQUEST_ID, UUID.randomUUID().toString());
+                .header(AUTHORIZATION, DEFAULT_AUTHORIZATION)
```

Comment:
Please restore indentation, the idea is to have:
```
given()
   .something()
when()
   .another()
....
```

Reply:
ok